screenly-cast/bin/install-wp-tests.sh

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
+
+if [ $# -lt 3 ]; then
+	echo "usage: $0 <db-name> <db-user> <db-pass> [db-host] [wp-version] [skip-database-creation]"
+	exit 1
+fi
+
+DB_NAME=$1
```

Comment:
Needs to be quoted.

Reply:
Okay.

Reply:
@vpetersson do the files look good? Have you had a chance to run a test?

Reply:
They should automatically run when a PR is opened, but it doesn't fire properly for some reason.